packages/next/next-server/server/render.tsx

```diff
@@ -357,7 +360,10 @@ export async function renderToHTML(
     }
   }
   if (isAutoExport) renderOpts.autoExport = true
-  if (isSpr) renderOpts.nextExport = false
+  if (isSpr) {
+    renderOpts.nextExport = false
+    renderOpts.isSSG = true
```

Comment:
Can we use the existing `__NEXT_SPR` on the page instead of adding something new? This data isn't necessary for initial hydration, so I think we can wait to grab it.

Reply:
Updated